kolibri/plugins/video_mp4_render/assets/src/vue/index.vue

```diff
@@ -1,8 +1,10 @@
 <template>
 
-  <div ref="videowrapperwrapper" class="videowrapperwrapper">
-    <loading-spinner v-if="loading"/>
```

Comment:
looks like this was changed from `v-if` to `v-show`. Was this intentional?

It _might_ have an effect on the way that the `loading-spinner` delay works, because it triggers based on the `mounted` event.